lib/shelley/test/integration/Main.hs

```diff
@@ -372,7 +374,9 @@ instance HasSeverityAnnotation TestsLog where
         MsgBaseUrl {} -> Notice
         MsgCluster msg -> getSeverityAnnotation msg
         MsgPoolGarbageCollectionEvent _ -> Info
-        MsgServerError{} -> Critical
+        MsgServerError e
+            | isAsyncException e -> Info
```

Comment:
Don't async exceptions also include `error` calls etc, besides the cancel thing?

Reply:
ErrorCall is synchronous (unless thrown with throwTo, but that would be silly).

Of the 4 AsyncException constructors, I would only be concerned with ThreadKilled and UserInterrupt.